Dockerfile

```diff
@@ -30,7 +30,6 @@ ADD composer.lock /app/composer.lock
 
 RUN COMPOSER_CACHE_DIR=/dev/null composer install --no-dev --no-autoloader
 
-ADD .git /app/.git
```

Comment:
Note: this is needed because of:

https://github.com/laminas/automatic-releases/blob/ea0062141bd984f60e4364600570487cec91b0d2/bin/console.php#L68

Without the git directory, the `ocramius/package-versions` plugin won't detect the correct version to display, which is quite interesting when people use `laminas/automatic-releases@v1` and want to know if something changed in the image.

I think the actual solution is to start publishing the docker images.

Reply:
Ah stupid `ocramius/package-versions` library TUT